### PR TITLE
add 4 firds

### PR DESCRIPTION
**Pull request by lotemh**
this feature is about adding some bird names to the text file

### CODE REVIEW

**Comment by Itay-Talias** [comment]
Hello World

File: hello.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-hi
\ No newline at end of file
+hi
+Nachlieli
+Sparrow
+Anafa
```
Comment (lotemh) [inline hello.txt:4]: Why did you add Anafa? It is not a very common bird.
Maybe add a dove instead?

File: hello.txt [hunk 2]
```diff
@@ -1 +1,5 @@
-hi
\ No newline at end of file
+hi
```
Comment (lotemh) [inline hello.txt:1]: Maybe use Capital Letter?

**Review by lotemh — COMMENTED** [review]
This code is great, check out some small comments!

File: hello.txt (hunk 1 above)
Comment (lotemh) [reply hello.txt:4]: Actually Anafa is very common in Israel
